lib/server.js

```diff
@@ -80,16 +80,18 @@ function createServer(storinfo, log) {
 
     server.use(restify.plugins.queryParser());
 
-    server.get({ path: '/poll', name: 'GetSharks' }, poll);
+    server.get({ path: '/storagenodes', name: 'GetStorageNodes' },
+        getStorageNodes);
```

Comment:
Only a nit (i.e. you can skip if you like, I don't want to make work): Typically we've used the work "List" for plural endpoints:  VMAPI ListVms, IMGAPI ListImages, Muskie ListDirectory and ListJobs.